cluster-operator/src/main/java/io/strimzi/operator/cluster/model/VolumeUtils.java

```diff
@@ -405,4 +407,100 @@ private static String getVolumeNameHashStub(String name)   {
             throw new RuntimeException("Failed to get volume name SHA-1 hash", e);
         }
     }
+
+    /**
+     * Creates the Client Secret Volume
+     * @param tls    Kafka Client tls object
+     * @param volumeList    List where the volumes will be added
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     */
+    protected static void createSecretVolume(ClientTls tls, List<Volume> volumeList, boolean isOpenShift) {
+        createSecretVolume(tls, volumeList, isOpenShift, null);
+    }
+
+    /**
+     * Creates the Client Secret Volume
+     * @param tls    Kafka Client tls object
+     * @param volumeList    List where the volumes will be added
+     * @param isOpenShift   Indicates whether we run on OpenShift or not
+     * @param alias   Alias to reference the Kafka Cluster
+     */
+    protected static void createSecretVolume(ClientTls tls, List<Volume> volumeList, boolean isOpenShift, String alias) {
+        if (tls != null) {
+            List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
+
+            if (trustedCertificates != null && trustedCertificates.size() > 0) {
```

Comment:
If `VolumeUtils` is just providing static helper methods:

1. It should have a `private` constructor.
2. These methods shouldn't be `protected` access. Either package-private or `public`.